hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -512,6 +512,7 @@ public function upgradeBundle() {
     $this->upgrade_1030();
     $this->upgrade_1032();
     $this->upgrade_1033();
+    $this->upgrade_1036();
```

Comment:
There is an upgrader added for $this->upgrade_1034(); but not included here. Guess we might need to include if in use.